packages/shared/src/components/squads/settings/PermissionSection.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ReactElement, useState } from 'react';
+import { SquadSettingsSection } from './SquadSettingsSection';
+import { Radio } from '../../fields/Radio';
+import { SourceMemberRole } from '../../../graphql/sources';
+import { Typography, TypographyType } from '../../typography/Typography';
+
+interface PermissionSectionProps {
+  initialMemberPostingRole?: SourceMemberRole;
+  initialMemberInviteRole?: SourceMemberRole;
+}
+
+const memberRoleOptions = [
+  {
+    label: 'All members (recommended)',
+    value: SourceMemberRole.Member,
+  },
+  {
+    label: 'Only moderators',
+    value: SourceMemberRole.Moderator,
+  },
+];
+
+export function PermissionSection({
+  initialMemberInviteRole,
+  initialMemberPostingRole,
+}: PermissionSectionProps): ReactElement {
+  const [memberPostingRole, setMemberPostingRole] = useState(
+    () => initialMemberPostingRole || SourceMemberRole.Member,
+  );
+  const [memberInviteRole, setMemberInviteRole] = useState(
+    () => initialMemberInviteRole || SourceMemberRole.Member,
+  );
+
+  return (
+    <div className="mt-2 flex flex-col gap-4 rounded-16 border border-border-subtlest-tertiary">
+      <Typography
+        type={TypographyType.Body}
+        className="flex items-center border-b border-border-subtlest-tertiary p-4 font-bold"
```

Comment:
bold is already a option on `Typography`

Reply:
Right. Will update this one.